content/en/security/application_security/threats/add-user-info.md

```diff
@@ -712,6 +712,8 @@ The events that can be automatically detected are:
 
 ### Automatic user activity event tracking mode
 
+<div class="alert alert-warning"><strong>This is deprecated</strong> and only applies to older agent versions before x</div>
```

Comment:
`before x` - we should link to the compatibility page here: https://docs.datadoghq.com/security/application_security/enabling/compatibility/

As the version depends per-library, we can't list all of them here.